var/spack/repos/builtin/packages/libyogrt/package.py

```diff
@@ -52,9 +54,43 @@ def configure_args(self):
 
         sched = self.spec.variants['scheduler'].value
         if sched != "system":
-            args.append('--with-%s=%s' % (sched, self.spec[sched].prefix))
+            if sched == "lsf":
+                # The LSF library depends on a couple of other libraries,
+                # and running the build inside of spack does not find
+                # them, and the user has to add them when they want
+                # to use -lyogrt. If we explicitly tell it what libraries
+                # to use, the user does not need to specify them
+                args.append('--with-lsf')
+                args.append('LIBS=-llsf -lrt -lnsl')
+            else:
+                args.append('--with-%s=%s' % (sched, self.spec[sched].prefix))
 
         if '+static' in self.spec:
             args.append('--enable-static=yes')
 
         return args
+
+    def install(self, spec, prefix):
+        # Run standard Autotools install first, then create a yogrt.conf file
+        # for the selected scheduler
+
+        AutotoolsPackage.install(self, spec, prefix)
```

Comment:
Reading all that this function does, we can possibly avoid overriding `install` by:
```suggestion
    @run_after('install)
    def create_yogrt_conf(self):
        spec, prefix = self.spec, self.prefix
```

Reply:
This does seem more tidy.